charts/argo-cd/templates/redis/deployment.yaml

```diff
@@ -5,12 +5,7 @@ kind: Deployment
 metadata:
   name: {{ template "argo-cd.redis.fullname" . }}
   labels:
-    app.kubernetes.io/name: {{ include "argo-cd.name" . }}-{{ .Values.redis.name }}
-    helm.sh/chart: {{ include "argo-cd.chart" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/part-of: argocd
-    app.kubernetes.io/component: {{ .Values.redis.name }}
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.redis.name "name" .Values.redis.name) | nindent 4 }}
     app.kubernetes.io/version: {{ .Values.redis.image.tag | quote }}
 spec:
   selector:
```

Comment:
Changing this selector would need the `--force` flag during update. Therefore I did not change this.